examples/mineral-extract-sites-detection/config_trne.yaml

```diff
@@ -92,4 +92,4 @@ assess_detections.py:
   output_folder: .
   iou_threshold: 0.1
   area_threshold: 50       # area under which the polygons are discarded from assessment
-  metrics_method: micro-average   # 1: macro-average ; 3: macro-weighted-average ; 2: micro-average
+  metrics_method: macro-average   # 1: macro-average ; 3: macro-weighted-average ; 2: micro-average
```

Comment:
It should make no difference with one class.

Reply:
Yes it is the same.